scripts/rollupCreation.ts

```diff
@@ -303,6 +303,8 @@ async function _getDevRollupConfig(
       loserStakeEscrow: ethers.constants.AddressZero,
       chainId: JSON.parse(chainConfig)['chainId'],
       chainConfig: chainConfig,
+      minimumAssertionPeriod: 75,
```

Comment:
should this be `0` since it's dev?

Reply:
I will leave it to have the same behavior as previously for now, not sure changing it will break testnode or sth